GarupaPico/GarupaPico/Model/Song.cs

```diff
@@ -4,7 +4,144 @@
 
 namespace GarupaPico.Model
 {
-    class Song
-    {
+    class Song {
+
+        /// <summary>
+        /// Gets song ID from the game.
+        /// </summary>
+        public int SongID { get; private set; }
+
+        /// <summary>
+        /// Gets the song name.
+        /// </summary>
+        public string SongName { get; private set; }
```

Comment:
Do not add unnecessary context to the object
`Song.SongName` is repeated. Use `Name` instead. The rest applies